src/StructuredLogger/BinaryLogger/BinLogReader.cs

```diff
@@ -77,14 +77,14 @@ public void Replay(Stream stream, Progress progress)
             // Use a producer-consumer queue so that IO can happen on one thread
             // while processing can happen on another thread decoupled. The speed
             // up is from 4.65 to 4.15 seconds.
-            var queue = new BlockingCollection<BuildEventArgs>(boundedCapacity: 5000);
-            var processingTask = System.Threading.Tasks.Task.Run(() =>
-            {
-                foreach (var args in queue.GetConsumingEnumerable())
-                {
-                    Dispatch(args);
-                }
-            });
+            // var queue = new BlockingCollection<BuildEventArgs>(boundedCapacity: 5000);
+            // var processingTask = System.Threading.Tasks.Task.Run(() =>
+            // {
+            //     foreach (var args in queue.GetConsumingEnumerable())
+            //     {
+            //         Dispatch(args);
+            //     }
+            // });

```

Comment:
Took these changes from the live branch as Browser doesn't support BlockingCollection.
Technically it could be replaced with AsyncCollection from AsyncEx library, but not sure what to do with it. 

Basically, this is the only one change that affects non-Avalonia versions. Even though looks insignificant.